### DISCUSSION TITLE
Incorporating Moderation Decisions into Records

### DISCUSSION THREAD

**Discussion by luoaide** [bluesky-social/atproto#165] [Protocol (atproto)]
Hello! I'm new here but extremely excited by the prospect of decentralized social media. The well-articulated division between speech and reach described in the Architecture doc may be our best hope at creating a fair, healthy, and democratic social media ecosystem.

One issue I foresee (and please forgive me if this is addressed somewhere): how does a user receive feedback about content moderation decisions made against their posts/comments/shares? When the big platforms make these decisions, they typically notify the impacted users with an email or a banner etc... This provides a nudge or social cue that perhaps the content of that post should be reconsidered. Given a healthy marketplace of services and labelers, one could expect to be 'excluded' from the timelines of at least a few other users. Fringe users could expect to be excluded from most timelines.

My question is how can we inject those important content decisions back into the original 'Record' and create the same cue that a Platform Ban, for example, achieves today?

Without direct feedback from 'Safety Labelers' or notification of the 'Actions' taken by Home timelines and specific Apps, I think we miss an opportunity. I fear that without such a mechanism, we encourage the unhealthy sorting of users along existing belief systems. This is just the start of any idea, and I'm curious how the creators would respond.

On a separate note, I'm surprised by the lack of discussion on these forums. I'm wondering if the creators expected a more vibrant community to debate these important decisions? This is such a cool and important project, I'm shocked that people aren't lining up to help build it! Count me in.

**Comment by pfrazee** [comment]
Hey @luoaide!

> how does a user receive feedback about content moderation decisions made against their posts/comments/shares? When the big platforms make these decisions, they typically notify the impacted users with an email or a banner etc... This provides a nudge or social cue that perhaps the content of that post should be reconsidered. Given a healthy marketplace of services and labelers, one could expect to be 'excluded' from the timelines of at least a few other users. Fringe users could expect to be excluded from most timelines.

That's a good question. It's hard to predict exactly how the system will play out. Will there be so many curated TLs and mod labelers that there's no point tracking? Will there be a few major players, but a long tail of players that generate a lot of noise? Should we be letting people know as often as possible, or only for the players they care about? Etc etc.

I'll keep an eye on the question, but we'll likely need to approach it iteratively and with feedback from people once the system is rolling out.

> My question is how can we inject those important content decisions back into the original 'Record' and create the same cue that a Platform Ban, for example, achieves today?

There are a couple of ways to address something like this. 1) The user's software can be crawling the network for these kinds of records and let the user know they found one. Subscribers to a labeler will automatically receive them. 2) Send notifications to the servers of users that might be interested. It depends on what we're trying to achieve, but both techniques are available to us.

> I fear that without such a mechanism, we encourage the unhealthy sorting of users along existing belief systems. This is just the start of any idea, and I'm curious how the creators would respond.

This is similar to the "Filter Bubble" debate. I think there's a legitimate concern that people could be given a sense that no other perspectives exist in the world, but we need to be sure we're giving people tools to see other perspectives and never attempting to control what they should be seeing. It's a difficult needle to thread, and I think we first need to understand how we create trust and authenticity in the network so that alternate perspectives can be effective.

> On a separate note, I'm surprised by the lack of discussion on these forums. I'm wondering if the creators expected a more vibrant community to debate these important decisions? This is such a cool and important project, I'm shocked that people aren't lining up to help build it! Count me in.

Love your enthusiasm! We haven't invited a ton of outside participation _yet_ and the reason is that we want to complete an initial build process first. We're still in an unstable dev phase where designs can shift, and it's terribly difficult on community contributors when a core team is making those kinds of changes. The quiet period we're in presently gives us space to move to a stable v1 as quickly as possible, and then we can engage more with the community.

**Reply by luoaide** [reply]
@pfrazee Thank you for the thoughtful response. The point on first understanding how to create trust and authenticity in the network is well taken. I'm excited to see this project continue to grow--I'll be following along!

**Reply by Cererean** [reply]
Presumably, there will be a record that a moderation service will have relating to each account or post they responded to? I think making this record public -- the post, the context, the report explaining the reasoning -- is an essential part of allowing subscribers to vet potential moderators. I know from Twitter that blocklists ostensibly for "transphobes" have a tendency to end up being filled with trans women the curator dislikes; making the reasoning public will allow people to avoid such moderators.
